internal/ingress/metric/collector/stats.go

```diff
@@ -50,6 +50,7 @@ type udpData struct {
 	Service   string `json:"service"`   // Label
 }
 
+// StatsCollector: Stores prometheus metrics and ingress meta-data
```

Comment:
No need of `:` in go :) `StatsCollector ...`

Reply:
😄